assets/javascripts/discourse/models/discourse-reactions-custom-reaction.js.es6

```diff
@@ -17,6 +21,29 @@ CustomReaction.reopenClass({
     ).catch(e => {
       bootbox.alert(`${e.jqXHR.status} ${e.errorThrown}`);
     });
+  },
+
+  findYourReactions(name, opts) {
+    opts = opts || {};
+
+    if(!name ) {
+      return;
+    }
+
+    const data = {};
+
+    if (opts.beforePostId) {
+      data.before_post_id = opts.beforePostId;
+    }
+
+    return ajax(`/discourse-reactions/posts/reactions-given/${name}.json`, { data }).then((posts) => {
```

Comment:
can you try to define and use adapter/store ?

Reply:
I tried to do this, but `this.store` is undefined here. Do you mean to remove this `findYourReactions` function from here and make an adapter instead?